docs/mctl_list_firmware.md

```diff
@@ -11,7 +11,12 @@ mctl list firmware [flags]
 ### Options
 
 ```
-  -h, --help   help for firmware
+      --component string   component type
```

Comment:
nit, maybe move this line under `-h`?

Reply:
this is auto generated docs, idk what i can do to change it

Reply:
Interesting. Thanks for letting me know!

Reply:
i do agree its awkward. It seems to be ordering it alphabetically.